### ISSUE TITLE
Flake8 warnings

### ISSUE THREAD

**Issue by parthea** [pydata/pandas-gbq#67]
We need to resolve these warnings to please `flake8`:

```
tony@tonypc:~/pydata-pandas-gbq$ flake8 pandas_gbq/
pandas_gbq/gbq.py:3:1: I100 Import statements are in the wrong order. import json should be before from datetime
pandas_gbq/gbq.py:5:1: I100 Import statements are in the wrong order. import uuid should be before from time
pandas_gbq/gbq.py:6:1: I100 Import statements are in the wrong order. import time should be before import uuid
pandas_gbq/gbq.py:7:1: I100 Import statements are in the wrong order. import sys should be before import time
pandas_gbq/gbq.py:11:1: I100 Import statements are in the wrong order. from distutils.version should be before import numpy
pandas_gbq/gbq.py:12:1: I101 Imported names are in the wrong order. Should be DataFrame, compat, concat
pandas_gbq/gbq.py:12:1: I201 Missing newline before sections or imports.
pandas_gbq/gbq.py:13:1: I101 Imported names are in the wrong order. Should be bytes_to_str, lzip
pandas_gbq/_version.py:183:5: N806 variable in function should be lowercase
pandas_gbq/_version.py:224:5: N806 variable in function should be lowercase
pandas_gbq/_version.py:226:9: N806 variable in function should be lowercase
pandas_gbq/tests/test_gbq.py:3:1: I100 Import statements are in the wrong order. import re should be before import pytest
pandas_gbq/tests/test_gbq.py:5:1: I201 Missing newline before sections or imports.
pandas_gbq/tests/test_gbq.py:6:1: I100 Import statements are in the wrong order. from time should be before import pytz
pandas_gbq/tests/test_gbq.py:6:1: I201 Missing newline before sections or imports.
pandas_gbq/tests/test_gbq.py:7:1: I100 Import statements are in the wrong order. import os should be before from time
pandas_gbq/tests/test_gbq.py:9:1: I100 Import statements are in the wrong order. import logging should be before from random
pandas_gbq/tests/test_gbq.py:15:1: I101 Imported names are in the wrong order. Should be range, u
pandas_gbq/tests/test_gbq.py:16:1: I101 Imported names are in the wrong order. Should be DataFrame, NaT
pandas_gbq/tests/test_gbq.py:16:1: I100 Import statements are in the wrong order. from pandas should be before from pandas.compat
pandas_gbq/tests/test_gbq.py:17:1: I201 Missing newline before sections or imports.
pandas_gbq/tests/test_gbq.py:18:1: I100 Import statements are in the wrong order. import pandas.util.testing should be before from pandas_gbq
pandas_gbq/tests/test_gbq.py:18:1: I201 Missing newline before sections or imports.
```

**Comment by jreback** [comment]
@parthea these don't show up with flake8 3.3

what version are you using?

**Comment by parthea** [comment]
I'm using flake8 v3.3 . The warnings just appeared all of a sudden and I'm not sure what changed. IIRC, @tswast  also saw this in #62 . It should be an easy fix to resolve it. I can do that now if we want to fix these.

```
tony@tonypc:~/pydata-pandas-gbq$ git status
HEAD detached at upstream/master
nothing to commit, working tree clean
tony@tonypc:~/pydata-pandas-gbq$ flake8 --version
3.3.0 (import-order: 0.12, mccabe: 0.6.1, naming: 0.4.1, pycodestyle: 2.3.1, pyflakes: 1.5.0) CPython 2.7.12 on Linux
tony@tonypc:~/pydata-pandas-gbq$ flake8 pandas_gbq/
pandas_gbq/gbq.py:3:1: I100 Import statements are in the wrong order. import json should be before from datetime
pandas_gbq/gbq.py:5:1: I100 Import statements are in the wrong order. import uuid should be before from time
pandas_gbq/gbq.py:6:1: I100 Import statements are in the wrong order. import time should be before import uuid
pandas_gbq/gbq.py:7:1: I100 Import statements are in the wrong order. import sys should be before import time
pandas_gbq/gbq.py:11:1: I100 Import statements are in the wrong order. from distutils.version should be before import numpy
pandas_gbq/gbq.py:12:1: I101 Imported names are in the wrong order. Should be DataFrame, compat, concat
pandas_gbq/gbq.py:12:1: I201 Missing newline before sections or imports.
pandas_gbq/gbq.py:13:1: I101 Imported names are in the wrong order. Should be bytes_to_str, lzip
pandas_gbq/_version.py:183:5: N806 variable in function should be lowercase
pandas_gbq/_version.py:224:5: N806 variable in function should be lowercase
pandas_gbq/_version.py:226:9: N806 variable in function should be lowercase
pandas_gbq/tests/test_gbq.py:3:1: I100 Import statements are in the wrong order. import re should be before import pytest
pandas_gbq/tests/test_gbq.py:5:1: I201 Missing newline before sections or imports.
pandas_gbq/tests/test_gbq.py:6:1: I100 Import statements are in the wrong order. from time should be before import pytz
pandas_gbq/tests/test_gbq.py:6:1: I201 Missing newline before sections or imports.
pandas_gbq/tests/test_gbq.py:7:1: I100 Import statements are in the wrong order. import os should be before from time
pandas_gbq/tests/test_gbq.py:9:1: I100 Import statements are in the wrong order. import logging should be before from random
pandas_gbq/tests/test_gbq.py:15:1: I101 Imported names are in the wrong order. Should be range, u
pandas_gbq/tests/test_gbq.py:16:1: I101 Imported names are in the wrong order. Should be DataFrame, NaT
pandas_gbq/tests/test_gbq.py:16:1: I100 Import statements are in the wrong order. from pandas should be before from pandas.compat
pandas_gbq/tests/test_gbq.py:17:1: I201 Missing newline before sections or imports.
pandas_gbq/tests/test_gbq.py:18:1: I100 Import statements are in the wrong order. import pandas.util.testing should be before from pandas_gbq
pandas_gbq/tests/test_gbq.py:18:1: I201 Missing newline before sections or imports.
```

**Comment by parthea** [comment]
I see the warnings here : https://travis-ci.org/tswast/pandas-gbq/jobs/245993675

**Comment by jreback** [comment]
these don't appear on master

**Comment by jreback** [comment]
https://travis-ci.org/pydata/pandas-gbq/jobs/250707426

**Comment by parthea** [comment]
My mistake. I had `flake8-import-order` installed locally which was generating these warnings. I've uninstalled `flake8-import-order`.

`flake8-import-order` is also automatically [installed in the conda environment ](https://travis-ci.org/tswast/pandas-gbq/jobs/245993675#L504) because it is listed as a dependency for the `pyjwt` package that is installed . I've created a PR to see if the `pyjwt` package needs `flake8-import-order`  ( conda-forge/pyjwt-feedstock/pull/5 ).
